.rubocop.yml

```diff
@@ -1,5 +1,5 @@
 AllCops:
-  TargetRubyVersion: 2.0
+  TargetRubyVersion: 2.1
```

Comment:
hm this strange, the one on master is 2.1 already.

https://github.com/phstc/shoryuken/blob/master/.rubocop.yml#L4

Is your branch updated with the latest master?

Reply:
My bad!